syncs/syncs.go

```diff
@@ -217,3 +217,18 @@ func (m *Map[K, V]) Range(f func(key K, value V) bool) {
 		}
 	}
 }
+
+// WaitGroup is identical to [sync.WaitGroup],
+// but provides a Go method to start a goroutine.
+type WaitGroup struct{ sync.WaitGroup }
+
+// Go calls the given function in a new goroutine.
+// It automatically increments the counter before execution and
+// automatically decrements the counter after execution.
+func (wg *WaitGroup) Go(f func()) {
```

Comment:
say that it must not be called concurrently with Wait.

Reply:
Done.